### PR TITLE
Pioneer 18d gliders

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP05MOAS-GL559/CP05MOAS-GL559_D00005_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00005/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00005/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00005/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00005/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00005/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-05-PARADM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP05MOAS-GL559/CP05MOAS-GL559_D00005_ingest.csv:6]: looks good

File: CP05MOAS-PG564/CP05MOAS-PG564_D00005_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00005/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-01-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00005/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00005/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-03-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_o.flort_o_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00005/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-04-FLORTO000,telemetered,Available,
+mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00005/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-05-NUTNRM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00005/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-06-PARADM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP05MOAS-PG564/CP05MOAS-PG564_D00005_ingest.csv:8]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: CP05MOAS-GL559/CP05MOAS-GL559_D00005_ingest.csv (hunk 1 above)
Comment (johnmlund) [reply CP05MOAS-GL559/CP05MOAS-GL559_D00005_ingest.csv:6]: looks good

File: CP05MOAS-PG564/CP05MOAS-PG564_D00005_ingest.csv (hunk 2 above)
Comment (johnmlund) [reply CP05MOAS-PG564/CP05MOAS-PG564_D00005_ingest.csv:8]: look ok

**Review by johnmlund — APPROVED** [review]

**Review by kuo-james — APPROVED** [review]

**Pull request merged**